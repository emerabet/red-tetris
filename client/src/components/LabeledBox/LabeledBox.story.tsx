import React from 'react';
import { storiesOf } from '@storybook/react';
import LabeledBox from './LabeledBox';

storiesOf('LabeledBox', module)
    .add('empty', () => (
        <LabeledBox
            label="room"
            content="super room"
        />
    ));
