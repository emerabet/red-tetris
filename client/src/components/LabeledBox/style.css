.outerLabel {
    border: dotted #f8b500 2px;
    padding: 10px;
    margin: 10px;
    width: 80%;
}