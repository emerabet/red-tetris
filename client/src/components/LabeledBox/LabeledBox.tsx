import React, { Component, useState, useEffect } from 'react';
import SquareLabel from '../Labels';
import RedTetris from '../RedTetris';

import './style.css';

interface GameProps {
  label?: string;
  content: string;
}

const LabeledBox: React.SFC<GameProps> = (props) => {
  return (
    <div className="outerLabel">
      <div>{props.label}</div>
      {/* {props.label !== undefined && <SquareLabel label={props.label} />} */}
      <div className="neon">{props.content}</div>
      {/* <SquareLabel label={props.content} red additionalClassName="paddingLabel"/> */}
    </div>
  );
};

export default LabeledBox;
