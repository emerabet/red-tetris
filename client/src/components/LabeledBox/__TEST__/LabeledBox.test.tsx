import React from 'react';

import renderer from 'react-test-renderer';
import LabeledBox from '../LabeledBox';

it('renders correctly with defaults', () => {
  const labeledBox = renderer.create(<LabeledBox
    label="room"
    content="super room"
     />).toJSON();
  expect(labeledBox).toMatchSnapshot();
});
