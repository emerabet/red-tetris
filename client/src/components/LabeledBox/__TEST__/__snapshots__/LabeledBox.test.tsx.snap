// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly with defaults 1`] = `
<div
  className="outerLabel"
>
  <div
    className="labelTitle "
  >
    <div>
       
      <div
        className="labelLetterM"
      >
        <div
          className="labelCube r1"
        />
        <div
          className="labelCube r2"
        />
        <div
          className="labelCube r3"
        />
        <div
          className="labelCube r4"
        />
        <div
          className="labelCube r5"
        />
        <div
          className="labelCube r6"
        />
        <div
          className="labelCube r7"
        />
        <div
          className="labelCube r8"
        />
        <div
          className="labelCube r9"
        />
        <div
          className="labelCube r10"
        />
        <div
          className="labelCube r11"
        />
        <div
          className="labelCube r12"
        />
      </div>
       
    </div>
    <div>
       
      <div
        className="labelLetterM"
      >
        <div
          className="labelCube o1"
        />
        <div
          className="labelCube o2"
        />
        <div
          className="labelCube o3"
        />
        <div
          className="labelCube o4"
        />
        <div
          className="labelCube o5"
        />
        <div
          className="labelCube o6"
        />
        <div
          className="labelCube o7"
        />
        <div
          className="labelCube o8"
        />
        <div
          className="labelCube o9"
        />
        <div
          className="labelCube o10"
        />
        <div
          className="labelCube o11"
        />
      </div>
       
    </div>
    <div>
       
      <div
        className="labelLetterM"
      >
        <div
          className="labelCube o1"
        />
        <div
          className="labelCube o2"
        />
        <div
          className="labelCube o3"
        />
        <div
          className="labelCube o4"
        />
        <div
          className="labelCube o5"
        />
        <div
          className="labelCube o6"
        />
        <div
          className="labelCube o7"
        />
        <div
          className="labelCube o8"
        />
        <div
          className="labelCube o9"
        />
        <div
          className="labelCube o10"
        />
        <div
          className="labelCube o11"
        />
      </div>
       
    </div>
    <div>
       
      <div
        className="labelLetter"
      >
        <div
          className="labelCube m1"
        />
        <div
          className="labelCube m2"
        />
        <div
          className="labelCube m3"
        />
        <div
          className="labelCube m4"
        />
        <div
          className="labelCube m5"
        />
        <div
          className="labelCube m6"
        />
        <div
          className="labelCube m7"
        />
        <div
          className="labelCube m8"
        />
        <div
          className="labelCube m9"
        />
        <div
          className="labelCube m10"
        />
        <div
          className="labelCube m11"
        />
        <div
          className="labelCube m12"
        />
        <div
          className="labelCube m13"
        />
      </div>
       
    </div>
  </div>
  <div
    className="labelTitle paddingLabel"
  >
    <div>
       
      <div
        className="labelLetterM"
      >
        <div
          className="labelCube redCube s1"
        />
        <div
          className="labelCube redCube s2"
        />
        <div
          className="labelCube redCube s3"
        />
        <div
          className="labelCube redCube s4"
        />
        <div
          className="labelCube redCube s5"
        />
        <div
          className="labelCube redCube s6"
        />
        <div
          className="labelCube redCube s7"
        />
        <div
          className="labelCube redCube s8"
        />
        <div
          className="labelCube redCube s9"
        />
        <div
          className="labelCube redCube s10"
        />
      </div>
       
    </div>
    <div>
       
      <div
        className="labelLetterM"
      >
        <div
          className="labelCube redCube u1"
        />
        <div
          className="labelCube redCube u2"
        />
        <div
          className="labelCube redCube u3"
        />
        <div
          className="labelCube redCube u4"
        />
        <div
          className="labelCube redCube u5"
        />
        <div
          className="labelCube redCube u6"
        />
        <div
          className="labelCube redCube u7"
        />
        <div
          className="labelCube redCube u8"
        />
        <div
          className="labelCube redCube u9"
        />
        <div
          className="labelCube redCube u10"
        />
      </div>
       
    </div>
    <div>
       
      <div
        className="labelLetterM"
      >
        <div
          className="labelCube redCube p1"
        />
        <div
          className="labelCube redCube p2"
        />
        <div
          className="labelCube redCube p3"
        />
        <div
          className="labelCube redCube p4"
        />
        <div
          className="labelCube redCube p5"
        />
        <div
          className="labelCube redCube p6"
        />
        <div
          className="labelCube redCube p7"
        />
        <div
          className="labelCube redCube p8"
        />
        <div
          className="labelCube redCube p9"
        />
        <div
          className="labelCube redCube p10"
        />
        <div
          className="labelCube redCube p11"
        />
      </div>
       
    </div>
    <div>
       
      <div
        className="labelLetterM"
      >
        <div
          className="labelCube redCube e1"
        />
        <div
          className="labelCube redCube e2"
        />
        <div
          className="labelCube redCube e3"
        />
        <div
          className="labelCube redCube e4"
        />
        <div
          className="labelCube redCube e5"
        />
        <div
          className="labelCube redCube e6"
        />
        <div
          className="labelCube redCube e7"
        />
        <div
          className="labelCube redCube e8"
        />
        <div
          className="labelCube redCube e9"
        />
        <div
          className="labelCube redCube e10"
        />
        <div
          className="labelCube redCube e11"
        />
        <div
          className="labelCube redCube e12"
        />
        <div
          className="labelCube redCube e13"
        />
      </div>
       
    </div>
    <div>
       
      <div
        className="labelLetterM"
      >
        <div
          className="labelCube redCube r1"
        />
        <div
          className="labelCube redCube r2"
        />
        <div
          className="labelCube redCube r3"
        />
        <div
          className="labelCube redCube r4"
        />
        <div
          className="labelCube redCube r5"
        />
        <div
          className="labelCube redCube r6"
        />
        <div
          className="labelCube redCube r7"
        />
        <div
          className="labelCube redCube r8"
        />
        <div
          className="labelCube redCube r9"
        />
        <div
          className="labelCube redCube r10"
        />
        <div
          className="labelCube redCube r11"
        />
        <div
          className="labelCube redCube r12"
        />
      </div>
       
    </div>
    <div>
       
      <div
        className="labelLetterS"
      />
       
    </div>
    <div>
       
      <div
        className="labelLetterM"
      >
        <div
          className="labelCube redCube r1"
        />
        <div
          className="labelCube redCube r2"
        />
        <div
          className="labelCube redCube r3"
        />
        <div
          className="labelCube redCube r4"
        />
        <div
          className="labelCube redCube r5"
        />
        <div
          className="labelCube redCube r6"
        />
        <div
          className="labelCube redCube r7"
        />
        <div
          className="labelCube redCube r8"
        />
        <div
          className="labelCube redCube r9"
        />
        <div
          className="labelCube redCube r10"
        />
        <div
          className="labelCube redCube r11"
        />
        <div
          className="labelCube redCube r12"
        />
      </div>
       
    </div>
    <div>
       
      <div
        className="labelLetterM"
      >
        <div
          className="labelCube redCube o1"
        />
        <div
          className="labelCube redCube o2"
        />
        <div
          className="labelCube redCube o3"
        />
        <div
          className="labelCube redCube o4"
        />
        <div
          className="labelCube redCube o5"
        />
        <div
          className="labelCube redCube o6"
        />
        <div
          className="labelCube redCube o7"
        />
        <div
          className="labelCube redCube o8"
        />
        <div
          className="labelCube redCube o9"
        />
        <div
          className="labelCube redCube o10"
        />
        <div
          className="labelCube redCube o11"
        />
      </div>
       
    </div>
    <div>
       
      <div
        className="labelLetterM"
      >
        <div
          className="labelCube redCube o1"
        />
        <div
          className="labelCube redCube o2"
        />
        <div
          className="labelCube redCube o3"
        />
        <div
          className="labelCube redCube o4"
        />
        <div
          className="labelCube redCube o5"
        />
        <div
          className="labelCube redCube o6"
        />
        <div
          className="labelCube redCube o7"
        />
        <div
          className="labelCube redCube o8"
        />
        <div
          className="labelCube redCube o9"
        />
        <div
          className="labelCube redCube o10"
        />
        <div
          className="labelCube redCube o11"
        />
      </div>
       
    </div>
    <div>
       
      <div
        className="labelLetter"
      >
        <div
          className="labelCube redCube m1"
        />
        <div
          className="labelCube redCube m2"
        />
        <div
          className="labelCube redCube m3"
        />
        <div
          className="labelCube redCube m4"
        />
        <div
          className="labelCube redCube m5"
        />
        <div
          className="labelCube redCube m6"
        />
        <div
          className="labelCube redCube m7"
        />
        <div
          className="labelCube redCube m8"
        />
        <div
          className="labelCube redCube m9"
        />
        <div
          className="labelCube redCube m10"
        />
        <div
          className="labelCube redCube m11"
        />
        <div
          className="labelCube redCube m12"
        />
        <div
          className="labelCube redCube m13"
        />
      </div>
       
    </div>
  </div>
</div>
`;
